usr/share/rear/layout/save/GNU/Linux/260_crypt_layout.sh

```diff
@@ -127,7 +168,12 @@ while read target_name junk ; do
         LogPrintError "Error: No 'uuid' value for LUKS$version volume $target_name in $source_device (mounting it or booting the recreated system may fail)"
     fi
 
-    echo "crypt /dev/mapper/$target_name $source_device type=$luks_type cipher=$cipher key_size=$key_size hash=$hash uuid=$uuid $keyfile_option" >> $DISKLAYOUT_FILE
+    (
+        echo -n "crypt /dev/mapper/$target_name $source_device type=$luks_type cipher=$cipher key_size=$key_size hash=$hash uuid=$uuid"
+        [ -n "$keyfile_option" ] && echo -n " $keyfile_option"
+        [ -n "$pbkdf_option" ] && echo -n " $pbkdf_option"
+        echo
+    ) >> $DISKLAYOUT_FILE
```

Comment:
I know we still use at many places a subshell
only to redirect output into a file
but usually this is not needed and
a compound group command is sufficient like
```
  { echo " ..."
    echo " ..."
  } 1>>$DISKLAYOUT_FILE
```
cf. https://github.com/rear/rear/pull/2947
so I would prefer `{ ... }` instead of a subshell
when only redirection is needed.